src/main.ts

```diff
@@ -0,0 +1,44 @@
+import { Octokit } from '@octokit/rest';
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+
+const githubToken = core.getInput('github_token');
+const autoTrigger = core.getInput('auto_trigger').toLowerCase() === 'true';
+const octokit = new Octokit({ auth: githubToken });
+
+async function run() {
+  try {
+    const context = github.context;
+    const payload = context.payload;
+
```

Comment:
It's fine to have 2 LB here, but I think we need to set up a Linter to maintain a consistent code structure across the entire project.